pyproject.toml

```diff
@@ -95,6 +96,7 @@ types-beautifulsoup4 = "~4.12.0.20240229"
 types-colorama = "~0.4.15.20240106"
 types-psutil = "~5.9.5.20240205"
 types-python-dateutil = "~2.9.0.20240316"
+types-cachetools = "^5.5.0.20240820"
```

Comment:
Could you please use "~=" version specifier in dev deps? And please put it in alphabetical order 🙂.

(We use `~=` for dev deps because we have a Renovate bot, which updates them to the next major/minor.)